src/Core/tests/DeviceTests/Handlers/CoreHandlerTestBase.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Microsoft.Maui.DeviceTests
 {
-	public abstract class CoreHandlerTestBase : HandlerTestBase, IDisposable
```

Comment:
(This was never needed because the base class already declared its disposableness.)